vcpkg/get_started/overview.md

```diff
@@ -4,12 +4,94 @@ description: Learn about how the vcpkg C/C++ package manager can help you acquir
 ms.topic: overview
 author: AugP
 ms.author: aupopa
-ms.date: 8/23/2023
+ms.date: 10/22/2023
 ---
+
 # vcpkg overview
 
-## What is vcpkg? 
+vcpkg (always spelled lowercase) is a free and open-source C/C++ package manager maintained by Microsoft and the C++ community. Initially launched in 2016 as a tool for assisting developers in migrating their projects to newer versions of Visual Studio, vcpkg has evolved into a cross-platform tool used by developers on Windows, macOS, and Linux with a large catalog of open-source libraries and enterprise-ready features. vcpkg is designed to not get in the way of your development process with support for a variety of build and project systems. As a C++ tool at heart, vcpkg is written primarily in C++ and CMake. It is designed to address common pain points for C/C++ developers in ways generally ignored by other package managers.
+
+## Why vcpkg?
+
+- Over 2200 open-source libraries to choose from, or bring your own
+- Consistent, cross-platform experience for Windows, macOS, and Linux
+- Dependencies can be built from source as needed with over 70 default build configurations (or customize your own builds)
+- Avoid ABI incompatibilities between packages and your consuming project
+- Prevent version conflicts and diamond problems in your dependency graph
+- Enterprise-ready tool relied by C/C++ developers across many industries, including: finance, gaming, embedded/IoT, semiconductors, defense, and at Microsoft
+- For MSBuild and CMake users: automatic integration with your build environment for a seamless dependency acquisition experience
+
+## vcpkg features
+
+### Ports and triplets
+
+A [port](../concepts/ports.md) in vcpkg is a versioned build recipe that produces a package consisting of a set of files. The most common package that can get produced when a port is installed is a C/C++ library consisting of headers, source code, and binaries.
+
+A [triplet](../users/triplets.md) captures the target build environment (cpu, os, compiler, runtime, etc.) in a single, convenient name. vcpkg provides over 70 triplets by default, but you can also define your own.
+
+To install a package on your system, vcpkg runs the script specified in the port's *portfile.cmake* file. This can include downloading source code and running a build. To run the build, it will use the triplet information to make sure that the final package matches the desired configuration.
```

Comment:
```suggestion
To install a package on your system, vcpkg runs the port's recipe file (a CMake script). The script may contain steps to download source code or run a build in your system. During builds, vcpkg uses the information in your triplet to make sure that the produced package matches your desired configuration.
```

Since this is an overview, I don't think we need to go into detail about `portfile.cmake`, just calling it a script should be enough.